src/app/models/workspace.server.model.ts

```diff
@@ -0,0 +1,55 @@
+import { Mongoose, Schema } from 'mongoose'
+
+import { IWorkspaceModel, IWorkspaceSchema } from '../../types'
+
+export const WORKSPACE_SCHEMA_ID = 'Workspace'
+
+const compileWorkspaceModel = (db: Mongoose): IWorkspaceModel => {
+  const WorkspaceSchema = new Schema<IWorkspaceSchema, IWorkspaceModel>({
+    title: {
+      type: String,
+      validate: [
+        /^[a-zA-Z0-9_\-./() &`;'"]*$/,
+        'Workspace title cannot contain special characters',
+      ],
+      required: 'Workspace title cannot be blank',
+      minlength: [4, 'Workspace title must be at least 4 characters'],
+      maxlength: [200, 'Workspace title can have a maximum of 200 characters'],
```

Comment:
I see you've followed the same format as form titles, but given the space constraints on the UI, maybe workspace titles should be shorter? what's the behaviour of the UI when the workspace title overflows?

this is a small but important decision because it's much much harder to make these limits more strict later on (what would we do for workspace titles that already exceed the limit?) than to make them more lenient

Reply:
Yep I think workspace titles can be shorter. I'll discuss this with design! 

As for the behaviour, when the workspace title overflows, the the title will be truncated and appended with ellipsis in the UI

Reply:
Reduced maximum title length to 50 characters as discussed!